### DISCUSSION TITLE
Active?

### DISCUSSION THREAD

**Discussion by icantremember** [Tunous/Dawn#494] [Q&A]
Hey there.. I was intro'd to this app on Fdroid (I think). I really like the swiping functionality. I see that the latest release even here on github is from back in Jan. Just wondering if you're actively developing. Thanks!

**Comment by Tunous** [comment]
Not actively developing (don't use Android device myself anymore), but I'm accepting contributions, doing code review and making releases if anything appears.

**Comment by AbsurdlySuspicious** [comment]
I use Dawn as main reddit client so I personally directly interested in contributing here, there's a lot of issues to address, but this is a matter of free time. Development is somewhat stale this year, but project is not abandoned.

**Comment by icantremember** [comment]
Thanks for the responses. I fully understand that something like this would not be a full time job for a lone dev.
Also thanks for the heads-up that your're no longer using Android. If I may ask, why bother continuing to support the app then? Surely some issue might come along that just isn't worth your time if you're not personally using the app.

**Reply by Tunous** [reply]
It's not like there is much to do, plus I'm not the only one with higher permissions in this repo anymore. If such an issue was to appear, in the worst case I would have to remove the app from the play store or give access to it to someone else. But since the app is in "alpha" mode in the store it is highly unlikely.
